src/main/java/de/tum/in/www1/artemis/service/theia/TheiaInfoContributor.java

```diff
@@ -0,0 +1,26 @@
+package de.tum.in.www1.artemis.service.theia;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_THEIA;
+
+import java.net.URL;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.info.Info;
+import org.springframework.boot.actuate.info.InfoContributor;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import de.tum.in.www1.artemis.config.Constants;
+
+@Profile(PROFILE_THEIA)
+@Component
+public class TheiaInfoContributor implements InfoContributor {
+
+    @Value("${theia.portal-url:}")
```

Comment:
```suggestion
    @Value("${theia.portal-url}")
```
The colon at the end sets an emty URL as default. Does this make sense?/Is this intended?
My assumption would be that a theia service should be required when starting Artemis with the `theia` profile.

Reply:
You are right there :) However, the server component does crash when trying to read the value from an empty configuration. It felt more robust to provide a fallback value and take care of empty variables later on (by disabling Theia altogether). Would you like me to remove the fallback, resulting in early crashes to resolve misconfiguration earlier?

Reply:
Yes, I think immediately crashing with the (automatic) error message that this value is required but missing then definitely makes sense.

Reply:
Done